uportal-war/src/main/java/org/jasig/portal/rest/SessionRESTController.java

```diff
@@ -81,6 +88,13 @@ public ModelAndView isAuthenticated(HttpServletRequest request, HttpServletRespo
             attributes.put("displayName", person.isGuest() ? "Guest" : person.getFullName());
             attributes.put("sessionKey", person.isGuest() ? null: key); //only provide keys to non guest users
             attributes.put("version", uPortalVersion);
+            if(swapperManager != null) {
```

Comment:
This if statement `should` always be true because of the `required = true` in the annotation. More of a paranoid programmer issue :).